jest.config.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+module.exports = {
+  clearMocks: true,
+  collectCoverage: true,
```

Comment:
```suggestion
  cacheDirectory: '.cache/jest',
  collectCoverage: true,
  coverageDirectory: './coverage',
  coverageReporters: ['html', 'json', 'text-summary'],
  testEnvironment: 'node',
  testRunner: 'jest-circus/runner',
```

Added back some settings we also override in renovate repo.